src/components/Match/MatchStory.jsx

```diff
@@ -540,6 +554,14 @@ const generateStory = (match) => {
   // Intro
   events.push(new IntroEvent(match));
 
+  // Prediction
+  match.players.forEach((player) => {
+    if (player.pred_vict === true) {
+      const order = player.player_slot < 5 ? player.player_slot - 89 : player.player_slot - 212;
```

Comment:
if i'm not mistaken it is literally the time of the event -- for example IntroEvent is set to -90, when players load into the game